spec/features/actors_feature_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+RSpec.describe "Actors", type: :feature do
+
+  before do
+    scorsese = create(:director, name: "Martin Scorsese", dob: "November 17, 1942")
+    departed = create(:movie, title: "The Departed", year: 2006, duration: 151, director: scorsese)
+    goodfellas = create(:movie, title: "Goodfellas", year: 1990, duration: 146, director: scorsese)
+
+    nolan = create(:director, name: "Christopher Nolan", dob: "July 30, 1970")
+    dark_knight = create(:movie, title: "The Dark Knight", year: 2008, duration: 152, director: nolan)
+    inception = create(:movie, title: "Inception", year: 2010, duration: 148, director: nolan)
+
+    leo = create(:actor, name: "Leonardo DiCaprio", dob: "November 11, 1974")
+    create(:character, name: 'Cobb', movie: inception, actor: leo)
+    create(:character, name: 'Billy Costigan', movie: departed, actor: leo)
+
+    jack = create(:actor, name: "Jack Nicholson", dob: "April 22, 1937")
+    create(:character, name: 'Frank Costello', movie: departed, actor: jack)
+
+    bob = create(:actor, name: "Robert De Niro", dob: "August 17, 1943")
+    create(:character, name: 'James Conway', movie: goodfellas, actor: bob)
+  end
+
+  context "index page" do
+    # TODO
+
+    it "displays each actor's name and dob", points: 5 do
+      visit "/actors"
+
+      actors = Actor.all
```

Comment:
again here, going on the philosophy that we want the students to be able to read the tests, if the students go to inspect the test, I think they'll understand what `Actor.all` even if they don't fully understand where the actor data came from, whereas `create_list` hides a lot.  Plus, if we go that route, we definitely need to use sequencing to avoid uniqueness validation errors.